packages/shared/src/graphql/squads.ts

```diff
@@ -53,7 +53,7 @@ type EditSquadOutput = {
 
 type PostToSquadProps = {
   id: string;
-  sourceId: string;
+  sourceId?: string;
```

Comment:
Is this one used anywhere still?

Reply:
this interface is shared between the create & update mutation. Left it as optional since it's needed on create